minui-list.c

```diff
@@ -1320,6 +1458,38 @@ bool parse_arguments(struct AppState *state, int argc, char *argv[])
         }
     }
 
+        // If we already have a title, don't overwrite it with the --title argument. Log an error and return false if both are provided
+    if (title_arg != NULL && strlen(state->title) > 0)
+    {
+        log_error("Both --header and --title arguments provided. Please use only one.");
+        return false;
+    }
+    else if (title_arg != NULL)
+    {
+        strncpy(state->title, title_arg, sizeof(state->title) - 1);
+    }
+
+    if (title_justification_arg != NULL)
+    {
+        // // The following block will cause the program to exit if the title justification is provided without a title
+        // // Leaving the block commented out reverts to not using the justification if there is no title
+        //
+        // // no justification without title
+        // if (strlen(state->title) == 0)
+        // {
+        //     log_error("WARNING: Title justification provided without a title. Please provide a title to justify.");
+        //     return false;
+        // }
```

Comment:
Just remove it. If there is no title and a justification is provided for the title, its the same as justifying nothing, so it's fine to not warn?